store/localcachelayer/post_layer.go

```diff
@@ -131,3 +141,25 @@ func (s LocalCachePostStore) GetPosts(options model.GetPostsOptions, allowFromCa
 
 	return list, err
 }
+
+func (s LocalCachePostStore) AnalyticsPostCount(options *model.PostCountOptions) (int64, error) {
+	if !options.AllowFromCache || options.MustHaveFile || options.MustHaveHashtag || !options.UsersPostsOnly || !options.ExcludeDeleted || options.TeamId != "" {
```

Comment:
Could we add a docstring above this function that says it only looks at the cache in the case `UsersPostsOnly` and `ExcludeDeleted` are not set?